berkeley-mobile/Common/EventTableViewCell.swift

```diff
@@ -99,13 +111,19 @@ class EventTableViewCell: UITableViewCell {
         let dateFormatter = DateFormatter()
         dateFormatter.dateFormat = "MM/dd/yyyy"
         eventTime.text = dateFormatter.string(from: entry.date)
-
+        
         eventCategory.text = type
         eventCategory.isHidden = type == nil
         eventCategory.backgroundColor = color
         eventTaggingColor.backgroundColor = color ?? Color.eventDefault
     }
     
+    func cellSetImage(image: UIImage, tapGesture: UITapGestureRecognizer) {
+        eventLogo.image = image
+        eventLogo.addGestureRecognizer(tapGesture)
```

Comment:
I'm not entirely sure about this, but if the cell gets dequeued for reuse, and this function is called again on a re-used cell, would we get a case where the logo has multiple gesture recognizers / would that be a problem?